UnitTests/ControllerTests/CreationControllerTests/GetVisualizationRulesAsyncActionTests.cs

```diff
@@ -289,7 +289,7 @@ public async Task GetVisualizationRulesAsync_Creates_TypeSpecificVisualizationRu
             ActionResult<VisualizationRules> result = await testController.GetVisualizationRulesAsync(rulesRequest);
 
             Assert.That(result.Value.VisualizationTypeSpecificRules, Is.Not.Null);
-            Assert.That(result.Value.VisualizationTypeSpecificRules.AllowShowingDataPoints, Is.True);
```

Comment:
I think more unit tests should be added for this change.

Reply:
Added tests that cover the valid data label use case and the now deprecated uses